### PR TITLE
AB-338: Create MusicBrainz schema in AcousticBrainz database

### PR DESCRIPTION
**Pull request by rsh7**
For implementing the method of copying the relevant information from MB database into a separate schema in AB database, this PR includes the addition of a new `musicbrainz` schema in AB and add a create table sql script.

### CODE REVIEW

**Comment by paramsingh** [comment]
rebase this please.

File: admin/sql/create_tables.sql [hunk 1]
```diff
@@ -154,4 +154,273 @@ CREATE TABLE feedback (
   suggestion         TEXT
 );
 
+
+CREATE SCHEMA IF NOT EXISTS musicbrainz;
```
Comment (paramsingh) [inline admin/sql/create_tables.sql]: this should be done in `create_musicbrainz_schema.sql`.

Comment (rsh7) [reply admin/sql/create_tables.sql]: Done!

**Comment by rsh7** [comment]
Added the new files for MusicBrainz schema, tables, indexes, primary and foreign keys and a function in manage.py to init the MB database along with creating the table structure and keys.

**Pull request merged**